security-admin/src/main/webapp/templates/common/ServiceManagerLayout_tmpl.html

```diff
@@ -39,16 +39,17 @@ <h3 class="wrap-header bold">{{tt 'h.serviceManager'}}</h3>
 									{{./this.attributes.name}} 
 									{{#if ../operation}} 
 									
-                                                                          <a href="javascript:void(0)" title="Export" type="button" class="download  pull-right text-decoration" data-id="downloadBtnOnService" data-servicetype="{{./this.attributes.name}}" style="margin-right: 5px;" >
+									<!-- Disable service addition/import/export ------
```

Comment:
We probably want to delete this instead of commenting it out. Also, in order to be able to submit a patch upstream this would need to be properly enabled in config flag so that it displays if not turned off... Do you have a good way to control this from ranger-admin code?